### DISCUSSION TITLE
Can DownloadHelper be detected?

### DISCUSSION THREAD

**Discussion by rewrecko** [aclap-dev/video-downloadhelper#251] [Help and Q&A]
### Your question

Hi , can downloading video with DownloadHelper be detected by organization / owners of content on Microsoft Strems?
Just out of curiosity :)

### Addon & CoApp version and details

All versions

**Comment by GrampaWildWilly** [comment]
Theoretically, yes, a site could determine you are downloading something.  But in practice, I doubt you'll encounter it.  When you play something in the player in a web page, the server streams the content to your client browser.  Typically, even a 4K video takes maybe 1 million byes per second of bandwidth, usually less.  Even TV livestreams usually come in comfortably under 1 million.  If a site notices that you are receiving data at 4 or 5 million bytes per second, the site could possibly say, "Oh look.  No human could possibly be watching something at that speed.  This guy must be downloading it."  Could.  Possibly.  I don't think you'll see this happen.  I don't think any site has bothered to do this kind of checking.  There are better ways to prevent downloading than looking at the data transfer speed.  Mind you, if the site is giving you service at that good level, they probably are perfectly happy to let you download whatever it is.  Could a site determine that you are using VDH specifically?  No.
